cs1/graphics.py

```diff
@@ -93,6 +98,7 @@ def clear_canvas():
   _check()
   global _fg
   _fg.clear()
+  _tb.put_token()
```

Comment:
This function is called by the UI to forward HTML canvas events. This is orthogonal.

Reply:
Nvm.